Library/Dnn.PersonaBar.UI/admin/personaBar/scripts/permissionGrid.js

```diff
@@ -361,10 +366,13 @@ if (typeof dnn.controls === "undefined" || dnn.controls === null) { dnn.controls
                 return false;
             }
 
+            var defaultPermission = this.getPermission('READ');

```

Comment:
Now I understand what @sleupold was saying. To select the correct permission you have to select by the combination of key and code. The `View folder` permission could be selected as

```suggestion
            var defaultPermission = this.getPermission({
                code: 'SYSTEM_FOLDER',
                key: 'READ'
            });
```